src/machine/arithmetic.rs

```diff
@@ -0,0 +1,97 @@
+use std::convert::TryFrom;
+
+use pijama_ast::{BinOp, BinOp::*, UnOp, UnOp::*};
+
+/// Trait determining how arithmetic operations should be handled.
+pub trait Arithmetic {
+    fn native_bin_op(op: BinOp, n1: i64, n2: i64) -> i64;
+    fn native_un_op(op: UnOp, n: i64) -> i64;
```

Comment:
Do we want to keep these names? I can't find better name right now.

Reply:
we can drop the `native_`. What do you think?

Reply:
Do we want to use the full name `binary_operation` and `unary_operation`?

Reply:
SGTM